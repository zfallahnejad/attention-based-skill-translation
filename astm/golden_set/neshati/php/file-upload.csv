187606,file-upload
1415724,file-upload
138475,file-upload
1226894,file-upload
476,file-upload
19068,file-upload
250259,file-upload
358679,file-upload
1353011,file-upload
