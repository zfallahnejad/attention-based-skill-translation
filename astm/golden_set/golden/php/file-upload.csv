187606,31,19,0.62
1415724,18,3,0.8571428571428571
138475,14,7,0.6666666666666666
1226894,11,4,0.7333333333333333
476,10,4,0.7142857142857143
19068,8,5,0.6153846153846154
250259,8,6,0.5714285714285714
358679,7,5,0.5833333333333334
1353011,7,7,0.5
